CMakeLists.txt

```diff
@@ -152,7 +153,7 @@ if(WIN32 AND ONEMKL_SYCL_IMPLEMENTATION STREQUAL "dpc++")
 endif()
 
 # Temporary disable sycl 2020 deprecations warnings for cuBLAS and cuSOLVER
-if(ONEMKL_SYCL_IMPLEMENTATION STREQUAL "dpc++" AND (ENABLE_CUBLAS_BACKEND OR ENABLE_CUSOLVER_BACKEND))
+if(ONEMKL_SYCL_IMPLEMENTATION STREQUAL "dpc++" AND (ENABLE_CUBLAS_BACKEND OR ENABLE_CUSOLVER_BACKEND OR ENABLE_ROCRAND_BACKEND))
```

Comment:
The deprecated warnings about using "hip" backend instead of "ext_oneapi_hip" could be suppressed. 
However, this will break the compilation when building with hipSYCL compiler.